src/items.zig

```diff
@@ -918,6 +923,7 @@ pub const Item = union(ItemType) { // MARK: Item
 			.tool => |_tool| {
 				zonObject.put("tool", _tool.save(allocator));
 			},
+			.null => {}, // TODO: should we handle this case?
```

Comment:
For now I'd say make it unreachable, to catch any use-cases that do not handle it correctly.